models/comments.model.js

```diff
@@ -28,5 +29,30 @@ exports.fetchArticleComments = (article_id) => {
         });
 };
 
+exports.addComment = (article_id, commentData) => {
+    const queryString = format(
+        'INSERT INTO comments (body, article_id, author, votes, created_at) VALUES (%L, %L, %L, %L, %L) RETURNING *;',
+        commentData.body,
+        article_id,
+        commentData.username,
+        0,
+        new Date(),
+    );
+    return db.query(queryString)
+        .then((result) => {
+            if (result.rows.length === 0) {
+                console.log('Hey buddy.. you fkd up');
```

Comment:
You can do a Promise.reject here, if you remove the check exists, as empty rows will mean article doesn't exist.